internal/olm/operator/bundle/install.go

```diff
@@ -72,7 +72,7 @@ func (i *Install) setup(ctx context.Context) error {
 	}
 
 	// Load bundle labels and set label-dependent values.
-	labels, bundle, err := operator.LoadBundle(ctx, i.BundleImage, i.SkipTLSVerify, i.UseHTTP)
+	labels, bundle, err := operator.LoadBundle(ctx, i.BundleImage, i.SkipTLS, i.SkipTLSVerify, i.UseHTTP)
```

Comment:
I would not pass SkipTLS to this call. Instead I would pass in true for both SkipTLSVerify AND UseHTTP *IF* SkipTLS was set.

Reply:
Removed from it.